controller/employeeController.js

```diff
@@ -1,187 +0,0 @@
-const companytable = require("../models").company;
-
-
-//for creating new user 
-const createemployee = async (req, res) => {
-    try {
-// yeh kiya hai dekhne ke liye ke koi field miss toh ni kar raha user dalna waise bhi yeh optional hai 
-        const { name, email, gender, mobile } = req.body;
-        if (!name || !email || !gender || !mobile) {
-          return res.status(400).json({
-            status: false,
-            message: "Missing required fields",
-          });
-        }
-      // Check if the email already exists in the database
-        const existingEmployee = await companytable.findOne({
-            where: { email: req.body.email }
-      });
-  //agar already exists karta hai toh 
-      if (existingEmployee) {
-        return res.status(409).json({
-          status: false,
-          message: "Email already exists",
-        });
-      }
-    else
-     { // Create a new employee if email doesn't exist
-      const newEmployee = await companytable.create({
-        name: req.body.name,
```

Comment:
const {name, email, gender, mobile} = req.body;

Destructing req body to get values would be a better way to do this, this works too